sogs/routes/rooms.py

```diff
@@ -296,11 +296,13 @@ def serve_file(room, fileId, filename):
     room_file = room.get_file(fileId)
     if not room_file:
         abort(http.NOT_FOUND)
+    if room_file.expiry == 0:
+        abort(http.NOT_FOUND)
```

Comment:
See my other comments: I think `model.Room` should be responsible for not returning an expired File rather than routes.